### PR TITLE
Add ESPN as a blockable service

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/244

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -929,6 +929,18 @@
...
     },
+    {
+      "id": "espn",
+      "name": "ESPN",
+      "rules": [
+        "||es.pn^",
+        "||espncdn.com^",
+        "||espn.com^",
+        "||espncricinfo.com^",
+        "||espn.net^"
```
Comment (zloyden) [inline assets/services.json:940]: I don't see the info about the `espn.net` and it's not opened on my end. Is it needed?

Comment (Sergey-Lyapin) [reply assets/services.json]: @zloyden

**Review by zloyden — APPROVED** [review]

**Comment by Alex-302** [comment]
News site. Not sure if it must be blocked.

**Comment by Sergey-Lyapin** [comment]
And also a streaming service
https://www.espn.com/watch/espnplus/?country=us&redirected=true

**Pull request merged**